config.yaml

```diff
@@ -45,12 +45,34 @@ mongo-dbname: workflows
 
 # mongo_password: secret
 
+# SMTP host
```

Comment:
Is there any reason why these parameters aren't included in the (`SMTPTask`) task definition?

Reply:
My idea is that smtp host/username/password are deployment settings, which you can easily override using env variables both in docker-compose and k8s. If we put those configuration in the definition of the tasks, they'll be built in the docker image making it impossible to change them at deploy time.

Am I wrong?

Reply:
You're probably right. I was just wondering if we're tailoring the task definition a little too much to our current setup/configs. However, I don't know if there will ever be use for configuring workflows with different smtp-hosts/e-mail accounts, but I just thought I'd to bring it up anyway.

Reply:
I believe it is not a wise choice to add deployment-specific settings to workflow definitions (SMTP settings, in this case).  I don't think we are going to use multiple SMTPs in the near future, so better to leave it as it is (SMTP settings from config/env variables) and manage it differently in the future when such a need arises.

Thanks for your review, anyway!